### PR TITLE
Document generate_interactive

### PR DESCRIPTION
**Pull request by juliohm**
Please consider adding this function to the top of the README. Most newcomers get lost with a manual creation of the template.

### CODE REVIEW

**Comment by juliohm** [comment]
Could you please review and merge this?

**Comment by CarloLucibello** [comment]
+1

**Comment by juliohm** [comment]
Is anyone monitoring this package? This change is trivial and yet it wasn't merged.

**Comment by CarloLucibello** [comment]
Maybe @oxinabox can help

File: README.md [hunk 1]
```diff
@@ -18,6 +18,18 @@ pkg> add PkgTemplates  # Press ']' to enter the Pkg REPL mode.
 
 ## Usage
 
+### TL;DR
```
Comment (oxinabox) [inline README.md:21]:
```suggestion
### Interactive Generation
```

**Review by oxinabox — APPROVED** [review]

**Comment by oxinabox** [comment]
Thanks, sorry this got lost.

**Pull request merged**